src/GraphQL/Execution/ExecutionResultJsonConverter.cs

```diff
@@ -78,8 +78,11 @@ private void WriteErrors(ExecutionErrors errors, JsonWriter writer, JsonSerializ
 
                 if (!string.IsNullOrWhiteSpace(error.Code))
                 {
+                    writer.WritePropertyName("extensions");
```

Comment:
Looking at this and the spec it appears that both `code` and `data` should be in the extensions.  Mind making that change?

Reply:
Done...